src/components/NavBar.js

```diff
@@ -0,0 +1,58 @@
+import { useState, useEffect } from "react";
+import { getUser } from "../api.js";
+import "../css/NavBar.css";
+import Button from "./Button";
+import Linkbrary from "../assets/Linkbrary.png";
+
+function NavBar() {
+  const [isLogin, setIsLogin] = useState(false);
+  const [userEmail, setUserEmail] = useState(null);
+  const [userImg, setUserImg] = useState(null);
+
+  const handleLogin = async () => {
+    try {
+      const response = await getUser();
+      const { email, ...rest } = await response;
```

Comment:
이미 getUser 함수를 호출할 때 await 했으니 다시 await 하지 않아도 됩니다!
```suggestion
      const { email, ...rest } = response;
```